packages/zoe/src/contractFacet/contractFacet.js

```diff
@@ -133,6 +135,12 @@ export function buildRootObject() {
           // invisibly-minted assets.
           E(zoeMintP).mintAndEscrow(totalToMint);
           zcfSeatAdmin.commit(seatStaging);
+          E(zoeInstanceAdmin).replaceAllocations([
+            {
+              zoeSeatAdmin,
+              allocation: zcfSeat.getCurrentAllocation(),
+            },
+          ]);
```

Comment:
can this repeated 6 line invocation be extracted to a function?  The parameters would only be zoeSeatAdmin and zcfSeat

Reply:
Done by Mark in his PR, thanks!